bit_manipulation/power_of_4.py

```diff
@@ -52,13 +52,9 @@ def power_of_4(number: int) -> bool:
     if number <= 0:
         raise ValueError("number must be positive")
     if number & (number - 1) == 0:
-        c = 0
-        while number:
-            c += 1
-            number >>= 1
-        return c % 2 == 1
-    else:
-        return False
```

Comment:
The function now doesn't return anything if the if-statement evaluates to False. This is why the build is failing.